src/main/java/com/apptware/interview/immutability/Student.java

```diff
@@ -1,15 +1,46 @@
 /** This class is expected to be immutable. Please make necessary changes. */
 package com.apptware.interview.immutability;
 
+import java.util.ArrayList;
 import java.util.Date;
 import java.util.List;
 import lombok.AllArgsConstructor;
 import lombok.Getter;
 
-@Getter
-@AllArgsConstructor
+//@Getter
```

Comment:
@getter
@AllArgsConstructor
both are not working so created manual all argument constructor setter and getter